### DISCUSSION TITLE
Centralized Docs Repo

### DISCUSSION THREAD

**Discussion by TheBoatyMcBoatFace** [bareboat-necessities/lysmarine_gen#270] [Ideas]
## Overview
As with many Github projects, Bareboat has grown to a vibrant community. During this growth, the docs have become difficult to navigate and search. 

I propose creating a new repository [bareboat-necessities/docs](https://github.com/bareboat-necessities/docs) where all docs and reference material can be consolidated and maintained. 

## Questions to Answer
- [ ] Where should new issues live?

## Problems this Solves
- Centralized location for documentation
- No out of date docs
- Single location for user reference

## Needed to Make this Happen
- [ ] Github Action to Pull Doc issues into this repo or push issues to new repo
- [ ] Docs SSG workflow _jenkins, gatsby, etc..._
- [ ] Redirect old docs to new docs

**Comment by TheBoatyMcBoatFace** [comment]
The current docs live in the [gh-pages branch of this repo](https://github.com/bareboat-necessities/my-bareboat/tree/gh-pages).

If we set up the docs repo, we will need to run a html-markdown conversion for the initial setup.

**Reply by mgrouch** [reply]
The docs are in central place already. The sources are in 

https://github.com/bareboat-necessities/my-bareboat/tree/master/docs

They are built by TravisCi

https://github.com/bareboat-necessities/my-bareboat/blob/master/.travis.yml

This script needs to migrate to github action as TravisCi is no longer free
